packages/nextjs/app/start/page.tsx

```diff
@@ -57,20 +63,23 @@ const StartLandingPage = () => {
             </p>
           </div>
         </div>
-        <div className="mb-8 mx-auto w-80 h-40 bg-[url('/assets/start/window-1.svg')] bg-center bg-no-repeat lg:mb-0 lg:absolute lg:right-[5%] lg:-bottom-20">
+        <div
+          className="mb-10 mx-auto w-80 h-40 bg-[url('/assets/start/window-1.svg')] bg-center bg-no-repeat lg:mb-0 lg:absolute lg:right-[5%] lg:-bottom-20"
+          style={windowSlideStyle}
+        >
```

Comment:
This slide in effect is pretty cool, but there are more modern ways to animate things. It would include adding a new library and it's harder to use, but the results can look a lot better.

Take a look at https://examples.motion.dev/ for some examples